demo/full/bxf-parser2/src/parser/node_commons/time.js

```diff
@@ -0,0 +1,26 @@
+import reduceChildren from "../reduce_children";
+import { parsedTimeCode } from "../../utils/timecode";
+
+export function parseSmpteTimeCode(node) {
+  return reduceChildren(node, (res, name, node) => {
+    switch (name.toLowerCase()) {
+      case "ns1:smptetimecode":
+        return node.textContent;
+    }
+    return res;
+  }, undefined);
+}
+
+export function parseStartDateTime(node) {
+  return reduceChildren(node, (res, name, node) => {
+    switch (name.toLowerCase()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "switch" statements should have at least 3 "case" clauses ([javascript:S1301](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS1301))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=366&issues=AWbt7Z4AW3Y69bz5aGXO&open=AWbt7Z4AW3Y69bz5aGXO)

[ISSUE_KEY]: AWbt7Z4AW3Y69bz5aGXO